src/sessions/SessionsSection.tsx

```diff
@@ -21,7 +29,15 @@ const Clients = (row: UserSessionRepresentation) => {
 };
 
 export const SessionsSection = () => {
+  const { t } = useTranslation("sessions");
   const adminClient = useAdminClient();
+  const [filterDropdownOpen, setFilterDropdownOpen] = useState(false);
+  const [filterType, setFilterType] = useState("All session types");
```

Comment:
Should this initial value not be replaced with `t("sessions:All session types")`? As the initial value might be mismatched if the user is using a different locale.

```suggestion
  const [filterType, setFilterType] = useState(t("sessions:All session types"));
```

Reply:
This is not fully implemented yet, and I followed the logic in KeysListTab, but I'm happy to change to useState(t("sessions:All session types")); for now.

Reply:
Done